Set_Demo/Card.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Card.swift
+//  Set_Demo
+//
+//  Created by Or Peleg on 24/04/2022.
+//
+
+import UIKit
```

Comment:
@or-peleg-healthy `Card` is a model object. It should not contain UI related entities.

Reply:
Changed import to Foundation. correct?